_docs/concepts/index.md

```diff
@@ -10,4 +10,5 @@ type: markdown
 
 ---
 
+{% assign docs = site.docs %}
```

Comment:
what does this do / why is it needed ? what happens if you missed one ?

Reply:
Turns the sidebar functionality into a function. Whereas before, the sidebar was closely coupled with our "docs" tree, it's now more general-purpose and can be used with any hierarchy of documents (such as blogs).

If you omit this statement, you'll end up with a blank sidebar.